ksql-engine/src/test/java/io/confluent/ksql/EndToEndEngineTestUtil.java

```diff
@@ -1070,11 +1070,8 @@ static Object avroToValueSpec(final Object avro,
       case STRING:
         return avro.toString();
       case ARRAY:
-        if (schema.getElementType().getName().equals(AvroData.MAP_ENTRY_TYPE_NAME) ||
-            Objects.equals(
-                schema.getElementType().getProp(AvroData.CONNECT_INTERNAL_TYPE_NAME),
-                AvroData.MAP_ENTRY_TYPE_NAME)
-            ) {
+        // TODO: figure out why this test is broken
```

Comment:
I had to make this change to get the build working locally. Not sure what I'm doing wrong yet. But this code in the master branch was causing this error when I build locally:

```
ksql-engine/src/test/java/io/confluent/ksql/EndToEndEngineTestUtil.java:[1075,57] cannot find symbol
[ERROR]   symbol:   variable CONNECT_INTERNAL_TYPE_NAME
[ERROR]   location: class io.confluent.connect.avro.AvroData
```

Reply:
This change was reverted. Still not sure what the issue was...